src/lib/expansion/index.ts

```diff
@@ -0,0 +1,41 @@
+import {NgModule, ModuleWithProviders} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
```

Comment:
I removed the import, but I still get the error. Maybe I'm not building material2 correctly.
But I did confirm that no other material2 module imports BrowserAnimationsModule.